### PR TITLE
Upgrading dependencies of this project

### PR DESCRIPTION
**Pull request by suztomo**

### CODE REVIEW

**Review by elharo — APPROVED** [review]

**Comment by suztomo** [comment]
ClassPathEntryTest.

```
Error:  Tests run: 9, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.01 s <<< FAILURE! - in com.google.cloud.tools.opensource.classpath.ClassPathEntryTest
Error:  testGetClassNames(com.google.cloud.tools.opensource.classpath.ClassPathEntryTest)  Time elapsed: 0.008 s  <<< ERROR!
org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact com.google.truth.extensions:truth-java8-extension:jar:1.1
	at com.google.cloud.tools.opensource.classpath.ClassPathEntryTest.resolveArtifact(ClassPathEntryTest.java:147)
	at com.google.cloud.tools.opensource.classpath.ClassPathEntryTest.testGetClassNames(ClassPathEntryTest.java:93)
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact com.google.truth.extensions:truth-java8-extension:jar:1.1
	at com.google.cloud.tools.opensource.classpath.ClassPathEntryTest.resolveArtifact(ClassPathEntryTest.java:147)
	at com.google.cloud.tools.opensource.classpath.ClassPathEntryTest.testGetClassNames(ClassPathEntryTest.java:93)

```

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathEntryTest.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 package com.google.cloud.tools.opensource.classpath;
 
+import static com.google.cloud.tools.opensource.dependencies.RepositoryUtility.CENTRAL;
```
Comment (elharo) [inline dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathEntryTest.java:19]: don't static import uncommon classes.

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathEntryTest.java [hunk 2]
```diff
@@ -143,6 +144,7 @@ private static Artifact resolveArtifact(String coordinates) throws ArtifactResol
 
     Artifact artifact = new DefaultArtifact(coordinates);    
     ArtifactRequest artifactRequest = new ArtifactRequest();
+    artifactRequest.addRepository(CENTRAL);
```
Comment (elharo) [inline dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathEntryTest.java:147]: why is this change needed? I'm surprised upgrading dependencies caused this.

**Review by elharo — CHANGES_REQUESTED** [review]

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathEntryTest.java (hunk 2 above)
Comment (suztomo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathEntryTest.java]: It has been broken in that every time we upgrade truth dependency, we had to upgrade this test case. For example: https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/1809#issuecomment-727011693 (November last year)
It's because `system.resolveArtifact` was not downloading the Truth artifact and the test case has been relying on local Maven repository to have the artifact.

**Comment by suztomo** [comment]
@elharo PTAL.

**Review by elharo — APPROVED** [review]
